settings.gradle

```diff
@@ -6,27 +6,35 @@ pluginManagement {
     }
 }
 
+// JitPack doesn't have credentials for the Yospace Maven repository.
+def hasYospace = System.getenv("YOSPACE_USERNAME") != null
+
 dependencyResolutionManagement {
     repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
     repositories {
         google()
         mavenCentral()
         maven { url 'https://maven.theoplayer.com/releases/' }
         maven { url 'https://raw.githubusercontent.com/NielsenDigitalSDK/nielsenappsdk-android/master/' }
-        maven {
-            url 'https://yospacerepo.jfrog.io/yospacerepo/android-sdk'
-            credentials {
-                username System.getenv("YOSPACE_USERNAME")
-                password System.getenv("YOSPACE_PASSWORD")
+        if (hasYospace) {
+            maven {
+                url 'https://yospacerepo.jfrog.io/yospacerepo/android-sdk'
+                credentials {
+                    username System.getenv("YOSPACE_USERNAME")
+                    password System.getenv("YOSPACE_PASSWORD")
+                }
             }
         }
     }
 }
 
 rootProject.name = "THEOplayer Connector"
-include ':app'
```

Comment:
This is not ideal, since now you **need** to have `YOSPACE_USERNAME` in your environment in order to see the `app` module.

I'm going to look for a better solution. But for now, this will do.